### PR TITLE
add doc for PR status checks [skip ci]

### PR DESCRIPTION
**Pull request by pxLi**
fix https://github.com/NVIDIA/spark-rapids/issues/2855, following-up of https://github.com/NVIDIA/spark-rapids/pull/3028

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -187,5 +187,18 @@ You can confirm that the update actually has happened by either inspecting its e
 `git diff` first or simply reexecuting `git commit` right away. The second time no file
 modification should be triggered by the copyright year update hook and the commit should succeed.
 
+### Pull request status checks
+A pull request should pass all status checks before merged.
+#### signoff check
+Please follow [sign-your-work](#sign-your-work) section, and make sure at least one commit in your pull request get signed-off.
```
Comment (jlowe) [inline CONTRIBUTING.md:193]:
```suggestion
Please follow the steps in the [Sign your work](#sign-your-work) section, and make sure at least one commit
in your pull request has a  sign-off.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -187,5 +187,18 @@ You can confirm that the update actually has happened by either inspecting its e
...
+#### blossom-ci
+The check runs on NVIDIA self-hosted runner, a [project committer](.github/workflows/blossom-ci.yml#L36) can manually trigger it by commenting `build`. It includes following steps,
+1. Mergeable check
+2. Blackduck vulnerability scan
+3. Fetch merged code (merge the pull request HEAD into BASE branch, e.g. fea-001 into branch-x)
+4. Run `mvn verify` and unit tests for multiple spark versions in parallel. Ref: [spark-premerge-build.sh](jenkins/spark-premerge-build.sh)
```
Comment (jlowe) [inline CONTRIBUTING.md:199]:
```suggestion
4. Run `mvn verify` and unit tests for multiple Spark versions in parallel. Ref: [spark-premerge-build.sh](jenkins/spark-premerge-build.sh)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -187,5 +187,18 @@ You can confirm that the update actually has happened by either inspecting its e
...
+
+If fail, you can click the `Details` link of this check, and go to `Post processing -> Start post processing` to find the uploaded log.
```
Comment (jlowe) [inline CONTRIBUTING.md:201]:
```suggestion
If it fails, you can click the `Details` link of this check, and go to `Post processing -> Start post processing` to find the uploaded log.
```

**Review by jlowe — COMMENTED** [review]
Please wrap lines to approx. 100 characters to match the style of the rest of the document.  It makes viewing it from a terminal nicer and more consistent, and it also helps localize comments on PRs.

Also do we have plans to change the "Post processing -> Start post processing" section to something with a much better name?  The fact that we have to document how to find logs after clicking the link is an indicator this could be much better.

**Comment by pxLi** [comment]
> Please wrap lines to approx. 100 characters to match the style of the rest of the document. It makes viewing it from a terminal nicer and more consistent, and it also helps localize comments on PRs.

sure, will reformat the change.
> Also do we have plans to change the "Post processing -> Start post processing" section to something with a much better name? The fact that we have to document how to find logs after clicking the link is an indicator this could be much better.

This is depends on blossom github support, we can file a feature request for them cc @zhanga5

**Comment by zhanga5** [comment]
> > Also do we have plans to change the "Post processing -> Start post processing" section to something with a much better name? The fact that we have to document how to find logs after clicking the link is an indicator this could be much better.
> 
> This is depends on blossom github support, we can file a feature request for them cc @zhanga5

OK, I will follow up this

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
build

**Review by jlowe — APPROVED** [review]

**Pull request merged**